logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,15 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
+    return false if @snapshot.nil?
+    if @snapshot_rotation_mutex.try_lock
+      has = @snapshot.metric_store.all.clone.map { |t| t.class.to_s }.include?("LogStash::Instrument::MetricType::Counter")
+      @snapshot_rotation_mutex.unlock
+      return has
+    end
+  end
```

Comment:
Also since all metrics are lazy created if you do `@snapshot.metric_store.all.size > 0` this will tell you if we have any metrics in the store.  

But lets create a predicated for it. `#empty?` `#present?` Naming is hard :)